clients/client-java/src/main/java/com/datastrato/gravitino/client/ObjectMapperProvider.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.client;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.cfg.EnumFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+/**
+ * Provides a singleton {@link ObjectMapper} configured for specific serialization and
+ * deserialization behaviors.
+ */
+public class ObjectMapperProvider {
```

Comment:
Can you please add a private constructor?

Reply:
done